Cargo.toml

```diff
@@ -36,6 +37,7 @@ pure-rust-locales = { version = "0.7", optional = true }
 rkyv = { version = "0.7", optional = true }
```

Comment:
Should this change the minimum version of rkyv to 0.7.41, the first version with `#[archive(check_bytes)]`?